docs/calamari/node-operators/join-network.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 3
+---
+
+# Join the network
+
+Specifically our test-net ----> https://telemetry.manta.systems/#/Calamari-Parachain%20Development
+
+## As a validator
+
+#### specific to this node
+  - ``my_node_name=my-awesome-manta-node``
+  - ``my_node_port=30333``
+  - ``my_node_ws_port=9944``
+  - ``my_db_path=/path/to/my/db``
+
+#### common for all nodes, you should have those files from the previous section
+  - ``relay_chain_spec_file=/full/path/to/rococo.json``
+  - ``para_chain_binary=/full/path/to/calamari-pc``
+
+```bash
+
+${para_chain_binary} \
+  --validator \
```

Comment:
this is just a nitpick since `--collator` and `--validator` have the same effect here, but i feel like the semantically correct flag is `--collator` on a parachain node.

Reply:
Cool.